README.md

```diff
@@ -15,8 +15,35 @@
 
 # <a name="overview"></a>Overview
 
-Each directory in this repository contains an automation workflow template for a specific community-built integration for Orbit workspaces.
+Inside this repository are template YAML workflow files you can use to automate **3rd party integrations** with your Orbit workspace.
 
+**What are 3rd party integrations?**
+
+All Orbit users have access to a host of [built-in integrations](https://orbit.love/integrations/), such as GitHub, Twitter, Slack, Discord and others. These built-in integrations can be added and set up from within the Orbit app.
```

Comment:
Should we refer to those as “first party” here, since we talk about “third party” in the rest of the README?

Reply:
I think I'd rather stick to "built-in" and "third party". Technically, "first party" might be more definitionally correct, but I think "built-in" conveys the intention of "first party" more clearly.